batch/scripts/report_status.sh

```diff
@@ -12,40 +12,38 @@ URL=${STATUS_URL}/${AWS_BATCH_JOB_ID}
 echo "URL: $URL"
 
 # Execute command, save the exit code and output (stdout AND stderr)
-OUTPUT=$("$@" 2>&1)
-EXIT_CODE=$?
+"$@" 2>&1 | tee output.txt
```

Comment:
This is namespaced, right? Somehow I can't spot the section where we create the work directory

Reply:
Re-wrote to use a file named after the batch id.